homeassistant/components/rejseplanen/sensor.py

```diff
@@ -51,9 +51,24 @@
         vol.All(cv.ensure_list, [cv.string]),
     vol.Optional(CONF_DEPARTURE_TYPE, default=[]):
         vol.All(cv.ensure_list,
-                [vol.In(list(['BUS', 'EXB', 'M', 'S', 'REG']))])
+                [vol.In(list([
+                    'BUS',
+                    'EXB',
+                    'TB',
+                    'LET',
+                    'M',
+                    'S',
+                    'REG',
+                    'IC',
+                    'LYN',
+                    'TOG'
+                ]))])
```

Comment:
Splat the lists together. No need to repeat them.

[*BUS_TYPES, *TRAIN_TYPES, *METRO_TYPES]

Reply:
I had a feeling there was a neater way of doing that. 👍